core/amber/src/main/scala/edu/uci/ics/amber/engine/common/storage/HDFSRecordStorage.scala

```diff
@@ -49,4 +49,8 @@ class HDFSRecordStorage[T >: Null <: AnyRef](hdfsLogFolderURI: URI)
     }
   }
 
+  override def containsFolder(folderName: String): Boolean = {
+    val path = folderPath.suffix("/" + folderName)
```

Comment:
can you use Path lib to do those operations so it may be better OS-independent? `"/"` is not supported for all OS.

Reply:
this is specific to HDFS.